web-animations-1/Overview.bs

```diff
@@ -1150,7 +1125,10 @@ as CSS Animations [[CSS-ANIMATIONS-1]].
     condition from the following, if any:
 
     <div class="switch">
-
+    :   If |animation|'s <a>current time</a> is <a>unresolved</a>,
+        <a>start time</a> is <a>unresolved</a>, the associated <a>timeline</a>
+        is finite and the <var>auto-rewind</var> flag is true
```

Comment:
Definition of finite timeline depends on #4862 and will be updated/referenced when the corresponding PR is merged.